mlir/lib/Dialect/Bufferization/Transforms/OneShotModuleBufferize.cpp

```diff
@@ -458,6 +458,15 @@ LogicalResult mlir::bufferization::bufferizeModuleOp(
       foldMemRefCasts(funcOp);
   }
 
+  // Bufferize all other ops.
+  for (Operation &op : moduleOp.getOps()) {
+    // Functions were already bufferized.
+    if (isa<func::FuncOp>(&op))
```

Comment:
Should this be on function like interface?

Reply:
For consistency reasons, `func::FuncOp` would be better here. Other function-like ops are not supported at the moment (they will fail to bufferize) and we exclusively use `func::FuncOp`, `func::CallOp`, `func::ReturnOp` in this file at the moment.